src/containers/logo/Logo.tsx

```diff
@@ -8,13 +8,24 @@ interface LogoProps extends BoxProps {
   light?: boolean
 }
 
-const Logo: FC<LogoProps> = ({ light = false, ...props }) => (
-  <Box
-    alt='logo'
-    component={ComponentEnum.Img}
-    src={light ? logoLight : logo}
-    {...props}
-  />
-)
+const Logo: FC<LogoProps> = ({ light = false, ...props }) => {
+  const handleClick = () => {
+    const welcomeSection = document.querySelector('[data-route="welcome"]')
```

Comment:
Using `querySelector` in a React application is not a best practice. Also take a look at the `HashLink` component.